### ISSUE TITLE
'succeed', 'precede', and 'surround' are undefined

### ISSUE THREAD

**Issue by sneakernets** [k0kubun/hamlit#22]
I'm trying to use `succeed` to make the full stops on my sentences not be bold, and not contain a space. This works in original haml.

Example code:
```haml
We need updated info from you on behalf of your organisation,
= succeed '.' do
  %strong= @organization.name
Use these forms below to attach updated info.
```

It is to look like this:

We need updated info from you on behalf of your organisation, **John Doe Industries**. Use these forms below to attach updated info.

Instead, I get 
```
NoMethodError - undefined method `succeed'
```

**Comment by k0kubun** [comment]
I'm sorry but currently some haml helpers are not available. It seems that [Hamlit::Helpers](https://github.com/k0kubun/hamlit/blob/master/lib/hamlit/helpers.rb) should have `succeed`, `precede` and `surround`, like [Haml::Helpers](https://github.com/haml/haml/blob/master/lib/haml/helpers.rb).
I'll fix it later. Thank you for using hamlit.

**Comment by sneakernets** [comment]
I have reformatted my haml syntax to not use them for now, but thank you!

**Comment by sneakernets** [comment]
I'll see what I can come up with in the meantime - are pull requests ok?

**Comment by k0kubun** [comment]
Any pull requests are welcome. Having proper tests for it would be better.
I can't work with this now because I'm busy until weekend, but I can review some pull requests.

**Comment by k0kubun** [comment]
Released as v1.2.0. Thank you :gem:

**Comment by sneakernets** [comment]
Oh thank you! :bento:
